recipe/meta.yaml

```diff
@@ -10,40 +10,45 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # patch to fix writing to the console in unicode in python2/3 friendly way
-    - console.patch
 
 build:
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   entry_points:
-    - py2neo = py2neo.cli.tool:main
-    - neokit = neokit:main
-    - geoff = py2neo.ext.geoff.__main__:main
+    - py2neo = py2neo.__main__:main
   number: 0
 
 requirements:
   build:
+    - click >=7.0
+    - colorama
+    - neobolt <1.8,>=1.7.3
+    - neotime <1.8,>=1.7.4
+    - prompt_toolkit <2.1,>=2.0.7
+    - pygments >=2.3.0,<2.4
+    - urllib3 <1.25,>=1.23
```

Comment:
I doubt you need most of these during the build phase.  You probably only need pip and python.